src/Trixi.jl

```diff
@@ -81,11 +81,19 @@ export initial_condition_constant,
        initial_condition_blob,
        initial_condition_orszag_tang,
        initial_condition_rotor,
-       initial_condition_briowu_shock_tube, initial_condition_torrilhon_shock_tube
+       initial_condition_briowu_shock_tube,
+       initial_condition_torrilhon_shock_tube,
+       initial_condition_ryujones_shock_tube,
+       initial_condition_shu_osher_shock_tube,
+       initial_condition_shu_osher_shock_tube_flipped
 
 export boundary_condition_periodic,
        boundary_condition_gauss,
-       boundary_condition_briowu_shock_tube, boundary_condition_torrilhon_shock_tube
+       boundary_condition_briowu_shock_tube,
+       boundary_condition_torrilhon_shock_tube,
+       boundary_condition_ryujones_shock_tube,
+       boundary_condition_shu_osher_shock_tube,
+       boundary_condition_shu_osher_shock_tube_flipped
```

Comment:
For setups coming with both BCs and ICs (and possibly sources), you can also put them together below (sorting by subject instead of by verb), e.g. after
```julia
export initial_condition_poisson_nonperiodic, source_terms_poisson_nonperiodic, boundary_condition_poisson_nonperiodic
```